tensorflow/lite/kernels/activations.cc

```diff
@@ -361,8 +381,18 @@ TfLiteStatus Relu1Eval(TfLiteContext* context, TfLiteNode* node) {
       }
       return kTfLiteOk;
     } break;
+    case kTfLiteUInt8: {
+      QuantizedRelu1<uint8_t>(input, output);
+      return kTfLiteOk;
+    } break;
+    case kTfLiteInt8: {
+      QuantizedRelu1<int8_t>(input, output);
+      return kTfLiteOk;
+    } break;
     default:
-      context->ReportError(context, "Only float32 supported currently, got %s.",
+      context->ReportError(context,
+                           "Only float32, kTfLiteUInt8, kTfLiteInt8 supported "
```

Comment:
please use uint8 and int8 instead of kTfLiteUint8.

Reply:
@suharshs , thanks for the review, i have updated the code as per your comments, kindly check and approve.

Regards
Amit